rules/eslint/es6/walmart.js

```diff
@@ -16,6 +18,8 @@ module.exports = {
     "constructor-super": 2,
     // enforce the spacing around the * in generator functions
     "generator-star-spacing": 2,
+    // disallow arrow functions where a condition is expected
+    "no-arrow-condition": 0,
```

Comment:
This is another ES6 gotcha. I'd love to turn it on: http://eslint.org/docs/rules/no-arrow-condition